src/utils/parsedJsonReplacer.ts

```diff
@@ -0,0 +1,16 @@
+const parsedJsonReplacer = (
+  key: string,
+  value: null | { id?: string; title: string }
+): any => {
```

Comment:
```suggestion
interface JsonValue {
  id?: string;
  title: string;
}

type ParsedJsonReplacerResult = JsonValue | string | null;

const parsedJsonReplacer = (
  key: string,
  value: JsonValue | null
): ParsedJsonReplacerResult => {
```

WDYT?

Not really sure for the name (`JsonValue`) but we can avoid the `any` =]

Reply:
Sure, it's better like this!